conda/testing/__init__.py

```diff
@@ -295,7 +295,6 @@ def __call__(
         """
         prefix = Path(prefix or self.path_factory())
 
-        reset_context([prefix / "condarc"])
```

Comment:
removed since this is already included as part of invoking the subcommands (see `context.__init__` calls with the parsed arguments)